docs/README.md

```diff
@@ -33,6 +33,35 @@ There are many cases where it should not contain the actual documentation:
   tooling, or recipes for using tooling which are common to us but too specific
   for general documentation.
 
+### Approach
+
+The runbook aims to be organised around the things it documents, rather than the
+organisational structure which may be responsible for that thing.
+This is a deliberate change from the previous structure, which had lead to a
+chicken-and-egg scenario of needing to know what something was and who was
+responsible for it and where the docs for it were in order to find the
+documentation for it.
+It is hoped that this approach will make the documentation more approachable to
+newcomers, for both reading and contributing.
+
+It's worth noting that, as of early 2020, the runbook is still a way off
+achieving this aim as it contains considerable left-over structure & content
+from the previous documentation which it subsumed.
+
+## Ownership
+
+While the runbook contains documentation relevant to the activities of the
+Competition and Kit Teams, neither Team owns it any more than any other
```

Comment:
More teams now exist. I know this is intended as the original model, so feel free to ignore. 

```suggestion
all SR teams, no one team owns it any more than any other
```

Reply:
Ah, good spot. This was written a long while back and I should have self-reviewed it!